responses/tests/test_responses.py

```diff
@@ -2035,6 +2035,36 @@ def run():
     assert_reset()
 
 
+def test_response_calls_and_registry_calls_are_equal():
+    @responses.activate
+    def run():
+        rsp1 = responses.add(responses.GET, "http://www.example.com")
+        rsp2 = responses.add(responses.GET, "http://www.example.com/1")
+        rsp3 = responses.add(
+            responses.GET, "http://www.example.com/2"
+        )  # won't be requested
```

Comment:
that comment is probably true only with the default registry, or?

Reply:
I mean, this mocked resource (`http://www.example.com/2`) isn't requested during the test to explain why we get empty `rsp3.calls`.
I can remove this if it confuses.

Reply:
that is fine to keep it for the testing purpose.

I just mean that if the user applies a different registry or we switch the default registry (which is unlikely), then this comment will not be correct

https://github.com/getsentry/responses#ordered-registry

Reply:
but this comment relates to the specific scope of the test. And is it relevant in this case?

If I were to initialize the test with
`@responses.activate(registry=OrderedRegistry)`

then it would be a different case with different expectations, wouldn’t it?

Reply:
that is more of a note rather than a call for an action :)

I am +1 with the current state of PR

now wait for @markstory to make his round of review